src/redux/actions/ResetPasswordAction.js

```diff
@@ -0,0 +1,47 @@
+import axios from "axios";
+import * as ActionTypes from "../action_types";
+import API_BASE_URL from "./index";
+
+/**
+ * error from forgot password request
+ *
+ * @param {string} error
+ * @return {object} type and error
+ */
+
+export const resetPasswordError = error => ({
+  type: ActionTypes.ERROR,
+  error
+});
+
+/**
+ * reset user password
+ *
+ * @param {string} responseData
+ * @return {object} type and responseData
+ */
+
```

Comment:
Attention to detail @meshack-mbuvi. When you receive feedback regarding one page, ensure you check all your other pages

Reply:
Point noted. I am fighting with backend bugs here but am managing